app-modules/etno/database/factories/ItemFactory.php

```
@@ -38,6 +39,25 @@ public function withDocumentOverrides(): static
         ]);
     }
 
+    public function published(bool $published = true): self
+    {
+        return $this->state(function (array $attributes) use ($published) {
+            $attributes['document_overrides'][] = 'access_rights';
+            $attributes['document_overrides'] = array_unique($attributes['document_overrides']);
+
+            $accessRights = fake()->randomElement(
+                $published
+                ? AccessRights::published()
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Metafori\Etno\Enums\AccessRights' in method 'published'. (undefined)

(StaticAccess)